package-lock.json

```diff
@@ -14169,7 +14169,8 @@
         "ansi-regex": {
```

Comment:
you should not push package-lock.json unless you modify package.json

Reply:
Reverted package-lock.json to development branch stage.